Dockerfile.base

```diff
@@ -124,7 +124,7 @@ FROM ci-desktop as ci-e2e
 
 # This chrome image is the latest version that accepts SameSite=None.
 # test/e2e/test/mocha.env.js will install a compatible chromedriver.
-ENV CHROME_VERSION="84.0.4147.135-1"
+ENV CHROME_VERSION="90.0.4430.93-1"
 ENV DETECT_CHROMEDRIVER_VERSION=true
 
 RUN wget --no-verbose https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb \
```

Comment:
Any reason we don't just use `https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb` ?

Reply:
Now that e2e works with latest Chrome I guess we can.